content/applications/hr/payroll.rst

```diff
@@ -357,3 +371,27 @@ Resume
    Currently, the :guilabel:`Resume` feature found inside the :menuselection:`Payroll app -->
    Configuration` menu is still in development and only serves a specific use case for Belgian
    markets. The documentation will be updated when this section has matured to more markets.
+
+Offers
```

Comment:
I don't see this in runbot 16.3

Reply:
Good to include sections so reader has direction

Reply:
Might be because this was in 16.4, not 16.3. I checked and I do see it in 16.4.

Reply:
@larm-odoo can you get clarity from support as to whether or not 16.4 is the stable release for 16 or if it is staging for 17? That will answer your question as to whether or not to include here for Odoo 16 doc.

Reply:
I asked Thoma (support person for payroll) and he said if you make a new trial database online (potential cusotmers0 they will be served a database in 16.4. If they create a database hosted on odoo SH they will be in version 16.0. So I believe form this info, that means we can use the 16.4 version, since new trials are all done in this.